.travis.yml

```diff
@@ -74,7 +74,7 @@ matrix:
         - make test-captplanet
 
     ### certificate signing tests ###
-    - env: MODE=integration
```

Comment:
Let's merge these tests together with captplanet tests:
```
    ### Integration tests ###
    - env: MODE=integration
      services:
        - redis
      install:
        - source scripts/install-awscli.sh
        - go install -race storj.io/storj/cmd/{captplanet,storj-sdk,storagenode,satellite,uplink,gateway}
      script:
        - make test-captplanet
        - make test-certificate-signing
```

Reply:
Also ensure that the `test-certificate-signing` is built using `-race`

Reply:
No objection here but do you care to explain why?

Reply:
Time needing to wait before somebody can merge something. Every travis job eats up the resources meaning with multiple concurrent PR-s building we need to wait that much longer.